src/Components/CollectionsHubRails/ArtistSeriesRail/__tests__/ArtistSeriesRail.test.tsx

```diff
@@ -41,15 +41,28 @@ describe("ArtistSeriesRail", () => {
     }
   })
 
-  it("Renders expected fields", () => {
+  it("showing the correct text, price guidance, and title", () => {
     const component = mount(<ArtistSeriesRail {...props} />)
     expect(component.text()).toMatch("Trending Artist Series")
-    expect(component.text()).toMatch("Flags unique collectins")
+    expect(component.text()).toMatch("Flags unique collections")
+    expect(component.text()).toMatch("From $1,000")
   })
 
-  it("No arrows when there are less than 5 collections", () => {
+  it("Does NOT show arrows when there are less than 5 collections", () => {
     const component = getWrapper()
-    expect(component.find(ArrowButton).length).toBe(1)
+    expect(component.find(ArrowButton).length).toBe(0)
+  })
+
+  it("Does NOT show arrows when there are exactly 4 collections", () => {
```

Comment:
This test and the one above it (`Does NOT show arrow when there are less than 5 collections`) seem redundant. I think we should remove one. 

Of the two, I prefer this test - I like seeing in the setup of the test that we're including 4 collections. But you're welcome to delete this one instead, if you prefer!